tests/Api/Admin/ShippingMethodsTest.php

```diff
@@ -97,7 +97,7 @@ public function it_archives_a_shipping_method(): void
     }
 
     /** @test */
-    public function it_restores_a_shipping_method(): void
```

Comment:
These were valid, why did you overwrite them?

Reply:
I dont know how it happened must admit 🤷🏽‍♂️